src/components/virtualRepeat/virtual-repeater.js

```diff
@@ -143,6 +143,7 @@ function VirtualRepeatContainerController(
       jWindow.off('resize', debouncedUpdateSize);
     });
 
+    $scope.$emit('$md-resize-enable');
```

Comment:
Is there a way to make this optional if you want your virtual-repeat to be totally static?